src/polyfills/browser-wasi.ts

```diff
@@ -1,48 +1,29 @@
-import { WASI, Fd, File, OpenFile, wasi } from '@bjorn3/browser_wasi_shim';
-import { type InternalWasi } from '../mod.ts';
-
-class Output extends Fd {
-  #mode: string;
-
-  constructor(mode: string) {
-    super();
-    this.#mode = mode;
-  }
-
-  fd_write(view8: Uint8Array, iovs: [wasi.Iovec]): { ret: number; nwritten: number } {
-    let nwritten = 0;
-    const decoder = new TextDecoder();
-    const str = iovs.reduce((acc, iovec, idx, all) => {
-      nwritten += iovec.buf_len;
-      const buffer = view8.slice(iovec.buf, iovec.buf + iovec.buf_len);
-      return acc + decoder.decode(buffer, { stream: idx !== all.length - 1 });
-    }, '');
-
-    (console[this.#mode] as any)(str);
-
-    return { ret: 0, nwritten };
-  }
-}
+import { WASI, Fd, File, OpenFile, ConsoleStdout } from '@bjorn3/browser_wasi_shim';
+import { type InternalWasi } from '../interfaces.ts';
 
 export async function loadWasi(
   _allowedPaths: { [from: string]: string },
   enableWasiOutput: boolean,
+  fileDescriptors: Fd[],
 ): Promise<InternalWasi> {
+  console.log('fileDescriptors = ', fileDescriptors);
```

Comment:
@chrisdickinson - does this `console.log` need to be here, or just leftover from debugging?